tests/test_sdr.py

```diff
@@ -27,11 +27,22 @@ def assert_expected_results_in_vector(expected_results, vector_path):
     watershed_results_layer = None
     watershed_results_feature = None
     for key in expected_results:
-        numpy.testing.assert_approx_equal(
-            actual_results[key], expected_results[key], significant=6,
-            err_msg=(
-                f'actual_results: {actual_results}, '
-                f'expected_results: {expected_results}'))
+        # numpy.testing.assert_almost_equal(
+        #     expected_results[key], actual_results[key], decimal=6)

```

Comment:
Is there a benefit to leaving these lines commented out?  If there isn't, could we remove them?